packages/ui/src/Button/index.tsx

```diff
@@ -1,7 +1,7 @@
 import { MouseEventHandler, useContext } from 'react';
 import styled, { css, DefaultTheme } from 'styled-components';
 import { asTransientProps } from '../utils/asTransientProps';
-import { Priority, ActionType, buttonInteractions } from '../utils/button';
+import { Priority, ActionType, focusOutline, overlays } from '../utils/button';
```

Comment:
Had to split `buttonInteractions` into `focusOutline` and `overlays`, since `Tabs` only uses the latter.